src/app/components/bibliography/bibliography.component.ts

```diff
@@ -0,0 +1,163 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { BibliographicCitation, BibliographyParserService } from '../../services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography',
+  templateUrl: './bibliography.component.html',
+  styleUrls: ['./bibliography.component.scss'],
+})
+export class BibliographyComponent implements OnDestroy {
+
+  public biblCits: BibliographicCitation[] = [];
+  private subscription: Subscription;
+  public isComplete: boolean;
+  selectedAlphOrder: 'Ascendent';
+  selectedField: 'Author';
+  selectedStyle: 'Chicago';
+  features: string[] = ['Author', 'Title', 'Date'];
```

Comment:
Why you named `features` the property in which you list available fields for sorting and `selectedField` for the property in which you store the selected one? You should uniform the names to make the code easier to understand.